### DISCUSSION TITLE
When using a shared internet document, images won't be displayed if you are not logged in.

### DISCUSSION THREAD

**Discussion by PenChien** [outline/outline#7063] [Self Hosting]
Hi,

When using a shared web document, images won't be displayed if you are **not** logged in.
I noticed that the URLs of images in a shared web document change after sharing, but I'm not sure if this is the cause of the issue.

For example:
Normal image URL is https://outline.example.com:9000
Shared web image URL is https://outline-bucket.example.com:9000

Thanks

**Comment by tommoor** [comment]
This is a configuration error of some sort, what does your env look like for storage related variables?

**Reply by PenChien** [reply]
docker.env
```
NODE_ENV=production
SECRET_KEY=ef3fbd29380d5b1dcb033814c4116f92da90d177b076b1f3988083ce98bbec09
UTILS_SECRET=ef3fbd29380d5b1dcb033814c4116f92da90d177b076b1f3988083ce98bbec09

DATABASE_URL=postgres://user:pass@getoutline_postgres_1:5432/outline
DATABASE_URL_TEST=postgres://user:pass@localhost:5432/outline-test
DATABASE_CONNECTION_POOL_MIN=
DATABASE_CONNECTION_POOL_MAX=PGSSLMODE=disable

REDIS_URL=redis://getoutline_redis_1:6379

URL=https://outline.example.com:9000
PORT=3000

COLLABORATION_URL=

AWS_ACCESS_KEY_ID=admin
AWS_SECRET_ACCESS_KEY=test123
AWS_REGION=local
#AWS_S3_ACCELERATE_URL=
AWS_S3_UPLOAD_BUCKET_URL=https://outline.example.com:9000
AWS_S3_UPLOAD_BUCKET_NAME=outline-bucket
AWS_S3_UPLOAD_MAX_SIZE=26214400
AWS_S3_FORCE_PATH_STYLE=false
AWS_S3_ACL=public-read
```

**Reply by tommoor** [reply]
I believe this is working as expected, for public shares we must return a "presigned" url for the image which includes the authentication. 

Try setting `AWS_S3_FORCE_PATH_STYLE=true`

**Reply by PenChien** [reply]
Works, thank you!!
